packages/devel/gperftools/package.mk

```
@@ -0,0 +1,14 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2021-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="gperftools"
+PKG_VERSION="gperftools-2.9.1"
```

```suggestion
PKG_VERSION="2.9.1"
```
I guess ?
https://github.com/gperftools/gperftools/releases/download/gperftools-2.9.1/gperftools-2.9.1.tar.gz
Hash could be different to your file.

---

I used a tag directly but if I apply url modification, this should work...